br/cmd/br/main_test.go

```diff
@@ -20,8 +20,39 @@ import (
 	"os"
 	"strings"
 	"testing"
+
+	"go.uber.org/goleak"
 )
 
+func TestMain(m *testing.M) {
+	skipLeakTest := false
+	var newArgs []string
+	for _, arg := range os.Args {
+		if arg == "--skip-goleak" {
+			skipLeakTest = true
+		} else {
+			newArgs = append(newArgs, arg)
+		}
+	}
+	os.Args = newArgs
```

Comment:
I will prefer make this flag an environment variable here. Which will be more flexible.

Reply:
I see no difference. It is just for test, no need to worried about performance or user friendly.
Also, there may be more than one `RUN_BR` command in one test. It is not so convenient to set and unset environment variables again and again.

Reply:
Well, in fact we can use `SOME_ENV=1 run_br` to set environment variable for one subprocess. Anyway this is only a test case, you may go whatever you like.